test/cbus_elixir_web/controllers/meeting_registration_controller_test.exs

```diff
@@ -0,0 +1,20 @@
+defmodule CbusElixirWeb.MeetingRegistrationControllerTest do
+  use CbusElixirWeb.ConnCase
+
+  test "GET /", %{conn: conn} do
+    conn = get(conn, meeting_registration_path(conn, :index))
+    assert html_response(conn, 200) =~ "Registration for"
+  end
+
+  test "Shows correct meeting day of", %{conn: conn} do
+    date = "2018-03-25"
+    conn = get(conn, "/?date=#{date}")
+    assert html_response(conn, 200) =~ "2018-04-03"
```

Comment:
This is the same test as below. One of them should check the actual meeting content is rendered

Reply:
Sure! What would you recommend, should I stub an attendee onto the meeting and test the UI for that attendee content? Change this test to something like "Shows correct meeting date with no attendees" and add a third test to test like "Shows correct meeting with attendee content when 1 or more attendees"?

Reply:
That sounds like an excellent test